[![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome)
[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](https://github.com/OmicsML/awesome-deep-learning-single-cell-papers/pulls)
# awesome-deep-learning-single-cell-papers

This repository keeps track of the latest papers on the single cell analysis with deep learning methods. We categorize them based on individual tasks.

We will try to make this list updated. If you found any error or any missed paper, please don't hesitate to open an issue or pull request.



- [Single Cell Book](#single-cell-book)
- [Single Cell RNA Tools](#single-cell-rna-tools)
- [Single Cell Visualization](#single-cell-visualization)
- [Benchmarking](#benchmarking)
- [Representation Learning](#representation-learning)
- [Pretrained Model](#pretrained-model)
- [Single Cell Atlas](#single-cell-atlas)
- [Batch Effect Correction](#batch-effect-correction)
- [Tumor Microenvironment-TME](#tumor-microenvironment-tme)
- [Cell-Cell Communication Events](#cell-cell-communication-events)
- [Gene Regulatory Network](#gene-regulatory-network)
- [Imputation](#imputation)
- [Spatial Domain](#spatial-domain)
- [Reference Embedding or Transfer Learning](#reference-embedding-or-transfer-learning)
- [Cell Segmentation](#cell-segmentation)
- [Cell Type Deconvolution](#cell-type-deconvolution)
- [Cell Type Annotation](#cell-type-annotation)
- [Cell Clustering](#cell-clustering)
- [Disease Prediction](#disease-prediction)
- [Multimodal Integration](#multimodal-integration)


## Single Cell Book
1. [[Single Cell Best Practices]](https://www.sc-best-practices.org/preamble.html)



## Single Cell RNA Tools 
[[Tool Summary]](https://www.scrna-tools.org/tools)

## Single Cell Visualization
1. [[Chanzuckerberg: An interactive explorer for single-cell transcriptomics data]](https://github.com/chanzuckerberg/cellxgene)
1. [[UCSC Cell Browser]](http://cells.ucsc.edu/)
1. [[Cytoscape]](https://cytoscape.org/)
1. [[UCSC Xena]](https://xena.ucsc.edu/)
1. [[ASAP: Automated Single-cell Analysis Pipeline]](https://asap.epfl.ch/)
1. [[GenePattern]](https://notebook.genepattern.org/)


## Benchmarking
1. [2022 Nature Methods] **Benchmarking atlas-level data integration in single-cell genomics** [[paper]](https://www.nature.com/articles/s41592-021-01336-8.pdf)
1. [2022 Nature Methods] **Benchmarking spatial and single-cell transcriptomics integration methods for transcript distribution prediction and cell type deconvolution** [[paper]](https://www.nature.com/articles/s41592-022-01480-9)
1. [2022 BioRxiv] **Benchmarking Automated Cell Type Annotation Tools for Single-cell ATAC-seq Data** [[paper]](https://www.biorxiv.org/content/10.1101/2022.10.05.511014v1)
1. [2022 Briefings in Bioinformatics] **Benchmarking methods for detecting differential states between conditions from multi-subject single-cell RNA-seq data** [[paper]](https://academic.oup.com/bib/article/23/5/bbac286/6649780)
1. [2021 Frontiers in Genetics] **Evaluating the Reproducibility of Single-Cell Gene Regulatory Network Inference Algorithms** [[paper]](https://www.frontiersin.org/articles/10.3389/fgene.2021.617282/full)
1. [2021 Nature Communications] **A benchmark study of simulation methods for single-cell RNA sequencing data** [[paper]](https://www.nature.com/articles/s41467-021-27130-w)
1. [2021 Genome Biology] **Benchmarking UMI-based single-cell RNA-seq preprocessing workflows** [[paper]](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02552-3)
1. [2020 Nature Methods] **Benchmarking algorithms for gene regulatory network inference from single-cell transcriptomic data** [[paper]](https://www.nature.com/articles/s41592-019-0690-6)
1. [2020 Genome Biology] **A benchmark of batch-effect correction methods for single-cell RNA sequencing data** [[paper]](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1850-9)
1. [2020 Nature Biotechnology] **A multicenter study benchmarking single-cell RNA sequencing technologies using reference samples** [[paper]](https://www.nature.com/articles/s41587-020-00748-9)
1. [2019 Nature Methods] **Benchmarking single cell RNA-sequencing analysis pipelines using mixture control experiments** [[paper]](https://www.nature.com/articles/s41592-019-0425-8)


## Representation Learning
1. [2023 ICLR under review] **scFormer: a universal representation learning approach for single-cell data using transformers** [[paper]](https://openreview.net/pdf?id=7hdmA0qtr5)
1. [2021 Current Opinion in Systems Biology] **Graph representation learning for single-cell biology** [[paper]](https://www.sciencedirect.com/science/article/pii/S2452310021000329)



## Pretrained Model
1. [2022 Nature Machine Intelligence] **scBERT as a large-scale pretrained deep language model for cell type annotation of single-cell RNA-seq data** [[paper]](https://www.nature.com/articles/s42256-022-00534-z)
1. [2021 PNAS] **Biological structure and function emerge from scaling unsupervised learning to 250 million protein sequences** [[paper]](https://www.pnas.org/doi/10.1073/pnas.2016239118)


## Single Cell Atlas
1. [[Cellxgene Datasets: 546 datasets by 2022]](https://cellxgene.cziscience.com/datasets)
1. [2022 Nature Methods] **Benchmarking atlas-level data integration in single-cell genomics** [[paper]](https://www.nature.com/articles/s41592-021-01336-8.pdf)
1. [2022 bioRxiv] **A unified analysis of atlas single cell data** [[paper]](https://www.biorxiv.org/content/10.1101/2022.08.06.503038v1.full.pdf)
1. [2022 Nature Biotechnology] **Integration of spatial and single-cell transcriptomic data elucidates mouse organogenesis** [[paper]](https://www.nature.com/articles/s41587-021-01006-2)
1. [2022 bioRxiv] **Supervised spatial inference of dissociated single-cell data with SageNet** [[paper]](https://www.biorxiv.org/content/10.1101/2022.04.14.488419v1)
1. [2022 Nature Communications] **Online single-cell data integration through projecting heterogeneous datasets into a common cell-embedding space** [[paper]](https://www.nature.com/articles/s41467-022-33758-z)

## Batch Effect Correction
1. [2020 Genomy Biology] **A benchmark of batch-effect correction methods for single-cell RNA sequencing data** [[paper]](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1850-9)
1. [2020 Nature Biotechnology] **A multicenter study benchmarking single-cell RNA sequencing technologies using reference samples** [[paper]](https://www.nature.com/articles/s41587-020-00748-9)
1. [2019 Nature Methods, **Harmony**] **Fast, sensitive and accurate integration of single-cell data with Harmony** [[paper]](https://www.nature.com/articles/s41592-019-0619-0)
1. [2018 Nature Biotechnology, **CCA**] **Integrating single-cell transcriptomic data across different conditions, technologies, and species** [[paper]](https://www.nature.com/articles/nbt.4096)
1. [2018 Nature Biotechnology, **Mutual Nearest Neighbors**] **Batch effects in single-cell RNA-sequencing data are corrected by matching mutual nearest neighbors** [[paper]](https://www.nature.com/articles/nbt.4091)
1. [2018 Nature Methods] **A test metric for assessing single-cell RNA-seq batch correction** [[paper]](https://www.nature.com/articles/s41592-018-0254-1)
1. [2017 Nature Biotechnology] **Multiplexed droplet single-cell RNA-sequencing using natural genetic variation** [[paper]](https://www.nature.com/articles/nbt.4042)

## Tumor Microenvironment-TME
1. [2022 Nature Biomedical Engineering] **Graph deep learning for the characterization of tumour microenvironments from spatial protein profiles in tissue specimens** [[paper]](https://www.nature.com/articles/s41551-022-00951-w)
1. [2022 Nature Communications] **SOTIP is a versatile method for microenvironment modeling with spatial omics data** [[paper]](https://www.nature.com/articles/s41467-022-34867-5)



## Cell-Cell Communication Events
1. [2022 bioRxiv] **scTensor detects many-to-many cell–cell interactions from single cell RNA-sequencing data** [[paper]](https://www.biorxiv.org/content/10.1101/2022.12.07.519225v1)
1. [2022 Nature Biotechnology] **Modeling intercellular communication in tissues using spatial graphs of cells** [[paper]](https://www.nature.com/articles/s41587-022-01467-z)
1. [2022 bioRxiv] **Decoding functional cell–cell communication events by multi-view graph
learning on spatial transcriptomics** [[paper]](https://www.biorxiv.org/content/10.1101/2022.06.22.496105v1)
1. [2021 Bioinformatics] **Identifying signaling genes in spatial single-cell expression data** [[paper]](https://pubmed.ncbi.nlm.nih.gov/32886099/)
1. [2020 Nature Methods] **NicheNet: modeling intercellular communication by linking ligands to target genes** [[paper]](https://www.nature.com/articles/s41592-019-0667-5)
1. [2020 Nature Communications] **Predicting cell-to-cell communication networks using NATMI** [[paper]](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7538930/pdf/41467_2020_Article_18873.pdf)
1. [2018 Nature] **Single-cell reconstruction of the early maternal–fetal interface in humans** [[paper]](https://www.nature.com/articles/s41586-018-0698-6)


## Gene Regulatory Network
1. [2022 Nature Biotechnology] **Multi-omics single-cell data integration and regulatory inference with graph-linked embedding** [[paper]](https://www.nature.com/articles/s41587-022-01284-4.pdf)
1. [2022 Biorxiv] **scMEGA: Single-cell Multiomic Enhancer-based Gene Regulatory Network Inference** [[paper]](https://www.biorxiv.org/content/10.1101/2022.08.10.503335v1)
1. [2022 Bioinformatics] **High-performance single-cell gene regulatory network inference at scale: the Inferelator 3.0** [[paper]](https://academic.oup.com/bioinformatics/article/38/9/2519/6533443)
1. [2022 Briefings in Bioinformatic] **SIGNET: single-cell RNA-seq-based gene regulatory network prediction using multiple-layer perceptron bagging** [[paper]](https://academic.oup.com/bib/article/23/1/bbab547/6484519)
1. [2020 Nature Methods] **Benchmarking algorithms for gene regulatory network inference from single-cell transcriptomic data** [[paper]](https://www.nature.com/articles/s41592-019-0690-6)
1. [2019 Genome Biology] **Single-cell transcriptomics unveils gene regulatory network plasticity** [[paper]](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1713-4)
1. [2017 Cell Syst] **Gene Regulatory Network Inference from Single-Cell Data Using Multivariate Information Measures** [[paper]](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5624513/)


## Imputation
1. [2018 Nature Communications] **An accurate and robust imputation method scImpute for single-cell RNA-seq data** [[paper]](https://www.nature.com/articles/s41467-018-03405-7)
1. [2019 Genome Biology] **DeepImpute: an accurate, fast, and scalable deep neural network method to impute single-cell RNA-seq data** [[paper]](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1837-6)
1. [2018 Cell] **Recovering Gene Interactions from Single-Cell Data Using Data Diffusion** [[paper]](https://www.cell.com/cell/fulltext/S0092-8674(18)30724-4)
1. [2018 Genome Biology] **VIPER: variability-preserving imputation for accurate gene expression recovery in single-cell RNA sequencing studies** [[paper]](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-018-1575-1)
1. [2021 PLOS Computational Biology] **G2S3: A gene graph-based imputation method for single-cell RNA sequencing data** [[paper]](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1009029)
1. [2021 Nature Communications] **scGNN is a novel graph neural network framework for single-cell RNA-Seq analyses** [[paper]](https://www.nature.com/articles/s41467-021-22197-x#Sec23)
1. [2021 iScience] **Imputing single-cell RNA-seq data by combining graph convolution and autoencoder neural networks** [[paper]](https://www.cell.com/iscience/fulltext/S2589-0042(21)00361-8)
1. [2022 PLOS ONE] **Single-cell specific and interpretable machine learning models for sparse scChIP-seq data imputation** [[paper]](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0270043)

## Spatial Domain
1. [2022 Nature Communications Biology] **Deciphering tissue structure and function using spatial transcriptomics** [[Review paper]](https://www.nature.com/articles/s42003-022-03175-5)
1. [2022 Genome Biology] **Statistical and machine learning methods for spatially resolved transcriptomics data analysis** [[Review paper]](https://genomebiology.biomedcentral.com/track/pdf/10.1186/s13059-022-02653-7.pdf)
1. [2022 Nature Communications] **Deciphering spatial domains from spatially resolved transcriptomics with adaptive graph attention auto-encoder** [[paper]](https://www.nature.com/articles/s41467-022-29439-6)
1. [2022 Nature Computational Science] **Cell clustering for spatial transcriptomics data with graph neural networks** [[paper]](https://www.nature.com/articles/s43588-022-00266-5)
1. [2022 Frontiers in Genetics] **Analysis and Visualization of Spatial Transcriptomic Data** [[paper]](https://arxiv.org/pdf/2110.07787.pdf)
1. [2021 Nature Methods] **SpaGCN: Integrating gene expression, spatial location and histology to identify spatial domains and spatially variable genes by graph convolutional network** [[paper]](https://www.nature.com/articles/s41592-021-01255-8)
1. [2021 Nature Biotechnology] **Spatial transcriptomics at subspot resolution with BayesSpace** [[paper]](https://www.nature.com/articles/s41587-021-00935-2)
1. [2021 Biorxiv] **Unsupervised Spatially Embedded Deep Representation of Spatial Transcriptomics** [[paper]](https://www.biorxiv.org/content/10.1101/2021.06.15.448542v2)
1. [2021 Genome Biology] **Giotto: a toolbox for integrative analysis
and visualization of spatial expression data** [[Tool]](https://genomebiology.biomedcentral.com/track/pdf/10.1186/s13059-021-02286-2.pdf)
1. [2021 Biorxiv] **Define and visualize pathological architectures of human tissues from spatially resolved transcriptomics using deep learning** [[paper]](https://www.biorxiv.org/content/10.1101/2021.07.08.451210v2)
1. [2020 Biorxiv] **stLearn: integrating spatial location, tissue morphology and gene expression to find cell types, cell-cell interactions and spatial trajectories within undissociated tissues** [[paper]](https://www.biorxiv.org/content/10.1101/2020.05.31.125658v1)
1. [2018 Nature Methods] **SpatialDE: Identification of Spatially Variable Genes** [[paper]](https://www.nature.com/articles/nmeth.4636)
1. [2018 Nature Biotechnology] **Identification of Spatially Associated Subpopulations by Combining scRNAseq and Sequential Fluorescence In Situ Hybridization Data** [[paper]](https://www.nature.com/articles/nbt.4260)
1. [2008 Journal of Statistical Mechanics] **Fast unfolding of community hierarchies in large networks** [[paper]](https://iopscience.iop.org/article/10.1088/1742-5468/2008/10/P10008)


## Reference Embedding or Transfer Learning
1. [2019 Nature Methods] **Data denoising with transfer learning in single-cell transcriptomics** [[paper]](https://www.nature.com/articles/s41592-019-0537-1)
1. [2018 Nature Methods] **Deep generative modeling for single-cell transcriptomics** [[paper]](https://www.nature.com/articles/s41592-018-0229-2)
1. [2020 Bioinformatics] **Conditional out-of-distribution generation for unpaired data using transfer VAE** [[paper]](https://academic.oup.com/bioinformatics/article/36/Supplement_2/i610/6055927?guestAccessKey=71253caa-1779-40e8-8597-c217db539fb5&login=false)
1. [2021 Nature Biotechnology] **Mapping single-cell data to reference atlases by transfer learning** [[paper]](https://www.nature.com/articles/s41587-021-01001-7)
1. [2021 Molecular Systems Biology] **Probabilistic harmonization and annotation of single-cell transcriptomics data with deep generative models** [[paper]](https://www.embopress.org/doi/full/10.15252/msb.20209620)
1. [2022 bioRxiv Preprint] **Biologically informed deep learning to infer gene program activity in single cells** [[preprint]](https://www.biorxiv.org/content/10.1101/2022.02.05.479217v2)

## Cell Segmentation
1. [2022 Cytometry A] **MIRIAM: A machine and deep learning single-cell segmentation and quantification pipeline for multi-dimensional tissue images** [[paper]](https://onlinelibrary.wiley.com/doi/epdf/10.1002/cyto.a.24541)[[code]](https://github.com/Coffey-Lab/MIRIAM)(MIRIAM)
1. [2021 Biorxiv] **Scellseg: a style-aware cell instance segmentation tool with pre-training and
contrastive fine-tuning** [[paper]](https://www.biorxiv.org/content/10.1101/2021.12.19.473392v1) [[code]](https://github.com/cellimnet/scellseg-publish)
1. [2021 Nature Biotechnology] **Cell segmentation in imaging-based spatial transcriptomics** [[paper]](https://www.nature.com/articles/s41587-021-01044-w) [[code]](https://github.com/kharchenkolab/Baysor)(Baysor)
1. [2021 Nature Biotechnology] **Whole-cell segmentation of tissue images with human-level performance using large-scale data annotation and deep learning** [[paper]](https://www.nature.com/articles/s41587-021-01094-0) [[code]](https://github.com/vanvalenlab/intro-to-deepcell)(Memser)
1. [2021 Nature Methods] **Cellpose: a generalist algorithm for cellular segmentation** [[paper]](https://www.nature.com/articles/s41592-020-01018-x) [[code]](https://www.github.com/mouseland/cellpose)(Cellpose)
1. [2021 Molecular Systems Biology]**Joint cell segmentation and cell type annotation for spatial transcriptomics** [[paper]](https://pubmed.ncbi.nlm.nih.gov/34057817/) [[code]](https://github.com/wollmanlab/JSTA) (JSTA)
1. [2020 Nature Communications]**A convolutional neural network segments yeast microscopy images with high accuracy** [[paper]](https://www.nature.com/articles/s41467-020-19557-4) [[code]](http://github.com/lpbsscientist/YeaZ-GUI)
1. [2020 Medical Image Analysis] **DeepDistance: A multi-task deep regression model for cell detection in inverted microscopy images** [[paper]](https://www.sciencedirect.com/science/article/abs/pii/S1361841520300840?via%3Dihub) (DeepDistance)
1. [2016 Computational Biology]**Deep Learning Automates the Quantitative Analysis of Individual Cells in Live-Cell Imaging Experiments** [[paper]](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005177) [[code]](https://github.com/vanvalenlab/deepcell-tf) (Deepcell)

## Cell Type Deconvolution
1. [2022 Biorxiv] **Accurate cell type deconvolution in spatial transcriptomics using a batch effect-free strategy** [[paper]](https://www.biorxiv.org/content/10.1101/2022.12.15.520612v1)
1. [2022 Nature Biotechnology] **Spatially informed cell-type deconvolution for spatial transcriptomics** [[paper]](https://www.nature.com/articles/s41587-022-01273-7#Sec2)
1. [2022 Nature Cancer] **Cell type and gene expression deconvolution with BayesPrism enables Bayesian integrative analysis across bulk and single-cell RNA sequencing in oncology** [[paper]](https://doi.org/10.1038/s43018-022-00356-3)
1. [2022 Nature Communications] **Advances in mixed cell deconvolution enable quantification of cell types in spatial transcriptomic data** [[paper]](https://www.nature.com/articles/s41467-022-28020-5#Sec2)
1. [2022 Nature Biotechnology] **Cell2location maps fine-grained cell types in spatial transcriptomics** [[paper]](https://doi.org/10.1038/s41587-021-01139-4)
1. [2021 Briefings in Bioinformatics] **DSTG: deconvoluting spatial transcriptomics data through graph-based artificial intelligence** [[paper]](https://doi.org/10.1093/bib/bbaa414)
1. [2021 Genome Research] **Likelihood-based deconvolution of bulk gene expression data using single-cell references** [[paper]](https://www.genome.org/cgi/doi/10.1101/gr.272344.120.)
1. [2021 Genome Biology] **SpatialDWLS: accurate deconvolution of spatial transcriptomic data** [[paper]](https://doi.org/10.1186/s13059-021-02362-7)
1. [2021 Nucleic Acids Research] **SPOTlight: seeded NMF regression to deconvolute spatial transcriptomics spots with single-cell transcriptomes** [[paper]](https://doi.org/10.1093/nar/gkab043)
1. [2021 Nature Biotechnology] **Robust decomposition of cell type mixtures in spatial transcriptomics** [[paper]](https://www.nature.com/articles/s41587-021-00830-w)
1. [2019 Nature Communications] **Accurate estimation of cell-type composition from gene expression data** [[paper]](https://doi.org/10.1038/s41467-019-10802-z)
1. [2019 Science] **Slide-seq: A scalable technology for measuring genome-wide expression at high spatial resolution** [[paper]](https://doi.org/10.1126/science.aaw1219)

## Cell Type Annotation 
1. [2022 Nature Method] **Annotation of spatially resolved single-cell data with STELLAR** [[paper]](https://www.nature.com/articles/s41592-022-01651-8) 
NOTE: annotated reference cell graph + query cell graph
1. [2022 Brief Bioinform] **scIAE: an integrative autoencoder-based ensemble classification framework for single-cell RNA-seq data** [[paper]](https://doi.org/10.1093/bib/bbab508)
1. [2022 Nature Communications] **scGCN is a graph convolutional networks algorithm for knowledge transfer in single cell omics** [[paper]](https://doi.org/10.1038/s41467-021-24172-y)
1. [2022 Science] **Cross-tissue immune cell analysis reveals tissue-specific features in humans** [[paper]](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7612735/#S9)
1. [2022 Bioinformatics] **CellMeSH: probabilistic cell-type identification using indexed literature** [[paper]](https://doi.org/10.1093/bioinformatics/btab834)
1. [2021 Nucleic Acids Research] **scDeepSort: a pre-trained cell-type annotation method for single-cell transcriptomics using deep learning with a weighted graph neural network** [[paper]](https://doi.org/10.1093/nar/gkab775)
1. [2021 BMC Bioinformatics] **Single-cell classification using graph convolutional networks** [[paper]](https://doi.org/10.1186/s12859-021-04278-2)
1. [2021 Genome Research] **Semisupervised adversarial neural networks for single-cell classification** [[paper]](https://genome.cshlp.org/content/31/10/1781)
1. [2020 BMC Bioinformatics] **EnClaSC: a novel ensemble approach for accurate and robust cell-type classification of single-cell transcriptomes** [[paper]](https://doi.org/10.1186/s12859-020-03679-z)
1. [2020 Bioinformatics] **ACTINN: automated identification of cell types in single cell RNA sequencing** [[paper]](https://doi.org/10.1093/bioinformatics/btz592)
1. [2020 Nature Communications] **SciBet as a portable and fast single cell type identifier** [[paper]](https://doi.org/10.1038/s41467-020-15523-2)
1. [2019 Nucleic Acids Research] **SuperCT: a supervised-learning framework for enhanced characterization of single-cell transcriptomic profiles** [[paper]](https://doi.org/10.1093/nar/gkz116)
1. [2019 Nucleic Acids Research] **CHETAH: a selective, hierarchical cell type identification method for single-cell RNA sequencing** [[paper]]( https://doi.org/10.1093/nar/gkz543)
1. [2019 Bioinformatics] **scMatch: a single-cell gene expression profile annotation tool using reference datasets** [[paper]](https://doi.org/10.1093/bioinformatics/btz292)
1. [2019 Cell Systems] **SingleCellNet: A Computational Tool to Classify Single Cell RNA-Seq Data Across Platforms and Across Species** [[paper]](https://doi.org/10.1016/j.cels.2019.06.004)
1. [2019 Genome Biology] **SingleCellNet: cPred: accurate supervised method for cell-type classification from single-cell RNA-seq data** [[paper]](https://doi.org/10.1186/s13059-019-1862-5)



## Cell Clustering
1. [2022 BMC Bioinformatics] **SC3s: efficient scaling of single cell consensus clustering to millions of cells** [[paper]](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-022-05085-z)
1. [2022 Bioinformatics] **GNN-based embedding for clustering scRNA-seq data** [[paper]](https://doi.org/10.1093/bioinformatics/btab787)
1. [2022 AAAI] **ZINB-based Graph Embedding Autoencoder for Single-cell RNA-seq Interpretations** [[paper]]( https://aaai-2022.virtualchair.net/poster_aaai5060)
1. [2022 Briefings in Bioinformatics] **Deep structural clustering for single-cell RNA-seq data jointly through autoencoder and graph neural network** [[paper]]( https://doi.org/10.1093/bib/bbac018)
1. [2022 Bioinformatics] **scGAC: a graph attentional architecture for clustering single-cell RNA-seq data** [[paper]]( https://doi.org/10.1093/bioinformatics/btac099)
1. [2022 Nature Computational Science] **Cell clustering for spatial transcriptomics data with graph neural networks** [[paper]]( https://www.nature.com/articles/s43588-022-00266-5)
1. [2021 Nature Communications] **Model-based deep embedding for constrained clustering analysis of single cell RNA-seq data** [[paper]]( https://www.nature.com/articles/s41467-021-22008-3)
1. [2020 NAR Genomics and Bioinformatics] **Deep soft K-means clustering with self-training for single-cell RNA sequence data** [[paper]]( https://doi.org/10.1093/nargab/lqaa039)
1. [2019 Nature Machine Intelligence] **Clustering single-cell RNA-seq data with a model-based deep learning approach** [[paper]]( https://www.nature.com/articles/s42256-019-0037-0)

<!---
## Cell Trajectory 
1. [2017 Nature Communications] **Reconstructing cell cycle and disease progression using deep learning** [[paper]](https://www.nature.com/articles/s41467-017-00623-3)
--->

## Disease Prediction
1. [2018 IJCAI] **Hybrid Approach of Relation Network and Localized Graph Convolutional Filtering for Breast Cancer Subtype Classification** [[paper]](https://www.ijcai.org/Proceedings/2018/490)
1. [2021 NPJ Digital Medicine] **DeePaN - A deep patient graph convolutional network integratingclinico-genomic evidence to stratify lung cancers benefiting from immunotherapy** [[paper]](https://www.nature.com/articles/s41746-021-00381-z)
1. [2022 Biocumputing] **CloudPred: Predicting Patient Phenotypes From Single-cell RNA-seq** [[paper]](https://www.worldscientific.com/doi/abs/10.1142/9789811250477_0031)
1. [2022 CHIL '20: Proceedings of the ACM Conference on Health, Inference, and Learning] **Disease state prediction from single-cell data using graph attention networks** [[paper]](https://dl.acm.org/doi/10.1145/3368555.3384449)

## Multimodal Integration
1. [2022 Genome Biology] **A benchmark study of deep learning-based multi-omics data fusion methods for cancer** [[Survey]](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-022-02739-2)
1. [2018 ICML] **MAGAN: Aligning biological manifolds** [[paper]](https://proceedings.mlr.press/v80/amodio18a.html)
1. [2019 PLoS computational biology] **Building gene regulatory networks from scATAC-seq and scRNA-seq using linked self organizing maps** [[paper]](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1006555)
1. [2020 Bioinformatics] **SCIM: universal single-cell matching with unpaired feature sets** [[paper]](https://academic.oup.com/bioinformatics/article/36/Supplement_2/i919/6055906)
1. [2021 Nature communications] **Multi-domain translation between single-cell imaging and sequencing data using autoencoders** [[paper]](https://www.nature.com/articles/s41467-020-20249-2)
1. [2020 PNAS] **BABEL enables cross-modality translation between multiomic profiles at single-cell resolution** [[paper]](https://www.pnas.org/doi/abs/10.1073/pnas.2023070118)
1. [2021 Genome biology] **Cobolt: integrative analysis of multimodal single-cell sequencing data** [[paper]](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02556-z)
1. [2021 Cell reports methods] **A mixture-of-experts deep generative model for integrated analysis of single-cell multiomics data** [[paper]](https://www.sciencedirect.com/science/article/pii/S2667237521001235)
1. [2021 Briefings in Bioinformatics] **Deep-joint-learning analysis model of single cell transcriptome and open chromatin accessibility data** [[paper]](https://academic.oup.com/bib/article/22/4/bbaa287/5985290)
1. [2021 Bioinformatics] **Deep cross-omics cycle attention model for joint analysis of single-cell multi-omics data** [[paper]](https://academic.oup.com/bioinformatics/article/37/22/4091/6283577)
1. [2022 Nature Biotechnology] **scJoint integrates atlas-scale single-cell RNA-seq and ATAC-seq data with transfer learning** [[paper]](https://www.nature.com/articles/s41587-021-01161-6)
1. [2022 Bioinformatics] **SMILE: mutual information learning for integration of single-cell omics data** [[paper]](https://academic.oup.com/bioinformatics/article/38/2/476/6384571)
1. [2022 SIGKDD] **Graph Neural Networks for Multimodal Single-Cell Data Integration** [[paper]](https://dl.acm.org/doi/10.1145/3534678.3539213)
1. [2022 Genome biology] **scDART: integrating unmatched scRNA-seq and scATAC-seq data and learning cross-modality relationship simultaneously** [[paper]](https://link.springer.com/article/10.1186/s13059-022-02706-x)
1. [2019 Biorxiv] **A Joint Model of RNA Expression and Surface Protein Abundance in Single Cells** [[paper]](https://www.biorxiv.org/content/10.1101/791947v1.abstract)
1. [2021 Biorxiv] **DeepMAPS: Single-cell biological network inference using heterogeneous graph transformer** [[paper]](https://www.biorxiv.org/content/10.1101/2021.10.31.466658v2)
1. [2022 Biorxiv] **Adaptative Machine Translation between paired Single-Cell Multi-Omics Data** [[paper]](https://www.biorxiv.org/content/10.1101/2021.01.27.428400v2)
1. [2022 Biorxiv] **Multigrate: single-cell multi-omic data integration** [[paper]](https://www.biorxiv.org/content/10.1101/2022.03.16.484643v1.abstract)
